<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>650</width>
     <height>650</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="foundPeaksCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Found Peaks</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fixHistogramCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>670</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fix Histogram</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>670</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Save for ML</string>
   </property>
   <widget class="QRadioButton" name="goodRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Good Image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="badRadioButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bad Image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dummyRadioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ignore Image</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
